js/capture.js

```diff
@@ -3,8 +3,16 @@
 * third-party requests to storage.
 */
 const capture = {
-  init() {
+  async init() {
     this.addListeners();
```

Comment:
I updated the code so that an error is printed to the console about the json file not being fetched properly, but the code still works (just sans whitelist).